### PR TITLE
Add OpenTitan Quickstart guide blog

### PR DESCRIPTION
**Pull request by GregAC**

### CODE REVIEW

File: site/content/blog/opentitan-quickstart-guide-video.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
++++
+Description = ""
+date = "2019-11-26T9:00:00+00:00"
+title = "OpenTitan Quickstart Guide video"
```
Comment (imphil) [inline site/content/blog/opentitan-quickstart-guide-video.md]: nit: consistent capitalization (and possibly a more exciting/actionable title, but I can't think of one either right now)

**Comment by GregAC** [comment]
How about this title 'Get started with OpenTitan'?

File: site/content/blog/get-started-with-opentitan.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
++++
+date = "2019-11-26T9:00:00+00:00"
+title = "Get started with OpenTitan"
+author = "Greg Chadwick"
++++
+
+
+Interested in trying out the recently announced OpenTitan? We've put together a video that goes through an overview of how the OpenTitan system (named Earl Grey) is put together and how to get up and running with our pre-built release (providing simulator binaries and pre-built FPGA images for the Nexys Video Artix-7 board). It follows the steps from the [OpenTitan Quickstart Guide](https://docs.opentitan.org/doc/ug/quickstart/).
```
Comment (asb) [inline site/content/blog/get-started-with-opentitan.md]: Either once sentence per line or wrap at 80 please.

Comment (asb) [inline site/content/blog/get-started-with-opentitan.md]: Should probably link OpenTitan.org and/or our announcement post

Comment (asb) [inline site/content/blog/get-started-with-opentitan.md]: Is it worth mentioning the "Earl Grey" name in this post? Even if so, maybe it's worth saying something like "The current OpenTitan prototype system" or similar?

**Comment by imphil** [comment]
deployed to production

**Pull request merged**